toonz/sources/toonzqt/camerasettingswidget.cpp

```diff
@@ -277,7 +282,8 @@ CameraSettingsWidget::CameraSettingsWidget(bool forCleanup)
       gridLay->addWidget(m_yPrev, 0, 4, Qt::AlignCenter);
 
       gridLay->addWidget(m_inchPrev, 1, 0, Qt::AlignRight | Qt::AlignVCenter);
-      gridLay->addWidget(new QLabel("Inch"), 1, 1,
+      QString units = Preferences::instance()->getCameraUnits();
+      gridLay->addWidget(new QLabel(units), 1, 1,
```

Comment:
It's a tiny UI problem so I think you can adjust after releasing this PR..

You set the current unit to the label of camera settings instead of "Inch". I think it is very nice modification itself.
But since the label is set once in the constructor, it remains unchanged if you change the camera unit.
Updating the label according to the camera unit or just setting more general name (such as "Size") would be OK.